Content.Shared/GameTicking/SharedGameTicker.cs

```diff
@@ -5,12 +5,14 @@
 using Robust.Shared.Serialization;
 using Robust.Shared.Serialization.Markdown.Mapping;
 using Robust.Shared.Serialization.Markdown.Value;
+using Robust.Shared.Timing;
 
 namespace Content.Shared.GameTicking
 {
     public abstract class SharedGameTicker : EntitySystem
     {
         [Dependency] private readonly IReplayRecordingManager _replay = default!;
+        [Dependency] private readonly IGameTiming _gameTiming = default!;
```

Comment:
I don't know how much this really matters, but this could be made `protected` and not duplicated in Server.